mineral-ui/mineral-ui
# [Table] New component

### Description

- Use a "future" enzyme adapter for React 16+ to accommodate use of create-react-context (polyfill for new-style React Context)
- Extract `rtlTextAlign` utility from Text and add spec
- Add support for additional types displayed under the component props in the website
- Add Table component
- Add examples and docs for Table
- Add spec and snapshots for Table

### Motivation and context

Connects #516 

### Screenshots, videos, or demo, if appropriate


https://516-table--mineral-ui.netlify.com/components/table

### How to test


1. `cd mineral-ui && npm start`
2. Navigate to Table
3. Confirm all looks and works well

### Types of changes

- New feature (non-breaking change which adds functionality)
- Other (provide details below)

Other: Everything that isn't directly Table-related in the description, above.

### Checklist




### How does this PR make you feel?

![Alan Rickman flipping a table](https://media.giphy.com/media/vwu9UTwIwFr7G/giphy.gif)
